config/cni/cilium/lb-ipam.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: "cilium.io/v2alpha1"
+kind: CiliumLoadBalancerIPPool
+metadata:
+  name: "lb-pool-1"
+spec:
+  cidrs:
+    - cidr: "172.18.250.0/24"
```

Comment:
@qasmi how did you choose the cidr ?

Reply:
To facilitate routing between the Docker layer and our laptop, it’s recommended to use a subnet from the CIDR used and routed by Docker (default 172.18.0.0/16)
If you run `docker inspect kind` you will see this in the bridge config:
<img width="580" alt="Screenshot 2024-04-25 at 18 08 36" src="https://github.com/fence-io/playground/assets/9462584/9d8b00d5-65f5-42d8-a5c2-2f0633db858e">